setup.py

```diff
@@ -15,16 +15,16 @@ def read(*parts):
     name='django-babel',
     description='Utilities for using Babel in Django',
     long_description=read('README.rst') + u'\n\n' + read('CHANGELOG.rst'),
-    version='0.6.3.dev0',
+    version='0.6.3',
     license='BSD',
     author='Christopher Grebs',
     author_email='cg@webshox.org',
-    maintainer='Thomas Grainger',
-    maintainer_email='django-babel@graingert.co.uk',
-    url='https://github.com/python-babel/django-babel/',
+    maintainer='Nikita Grygoriev',
```

Comment:
You are replacing the maintainer etc. here. This also changes the URL, not sure that is intended?